radsat-sk/framework/fileio/RFram.c

```diff
@@ -16,6 +16,13 @@
  * @brief Initialize the FRAM peripheral driver.
  */
 void framInit(void) {
+
+	uint16_t err = FRAM_start();
+
+	if (err != 0) {
+		return err;
+	}
+
 	FRAM_start();

```

Comment:
Your function would return NULL if the err == 0 because FRAM_start() is not in a return statement